website/docs/components/application-state/partials/code/component-api.md

```diff
@@ -0,0 +1,36 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="errorCode" @type="string">
+    The error code to be displayed.
+  </C.Property>
+  <C.Property @name="hasDivider" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Indicates if there should be a visible divider above the footer.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
```

Comment:
Don't forget to document that the Header, Body, and Footer also support ...attributes below.

Reply:
I wonder if we should? Since they are contextual components they're not relevant to the consumer, but only for our own test patterns. WDYT?

Reply:
Whatever we normally do is fine by me. If we normally wouldn't document then that's fine.

Reply:
My logic would be: 
If we think they are going to pull apart the pieces and make their own, prolly should document it. 
If they are contextual components and it wouldn't make sense for them to do that, and we've only added `...attributes` to make it easier to have testing conventions internally, it's more confusing to have it in the documentation. 

WDYT?